versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayParentUpdatesMojo.java

```diff
@@ -46,6 +60,78 @@ public class DisplayParentUpdatesMojo extends AbstractVersionsDisplayMojo {
 
     public static final int MESSAGE_LENGTH = 68;
 
+    // ------------------------------ FIELDS ------------------------------
+
+    /**
+     * <p>If {@code skipResolution} is not set, specifies the <em>bottom</em> version considered
+     * for target version resolution. If it is a version range, the resolved version will be
+     * restricted by that range.</p>
+     *
+     * <p>If {@code skipResolution} is {@code true}, will specify the target version to which
+     * the parent artifact will be updated.</p>
+     * @since 2.17.0
+     */
+    @Parameter(property = "parentVersion")
+    protected String parentVersion = null;
```

Comment:
Can be parameter fields `private`?

Reply:
My aim was to be as close to `UpdateParentMojo` as possible. All parameters are protected there as well. Should I change them to `private` visibility in both classes?

Reply:
Guilty as charged. Makes it easier to test indeed without the need to use reflection.

Reply:
So let's leave it like this?

Reply:
ok